src/main/java/edu/umich/its/spe/PersistTimestamp.java

```diff
@@ -33,85 +36,73 @@
 	@Autowired
 	private SPESummary spesummary;
 
-	private LocalDateTime startingTime = null;
-
-	// Format an internal timestamp into the expected string.
-	protected static String formatTimestamp(LocalDateTime ts) {
-		final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
-		String s = ts.format(formatter);
-		return s;
-	}
-
-	public String readLastGradeTransferTime() throws PersistBlobException {
+	public Instant readTestLastTakenTime() throws PersistBlobException {
 		String blob = persistString.readBlob();
+		blob = blob.trim();
```

Comment:
I feel the `blob` here represent a timestamp and the name is misleading

Reply:
Either "blob" or "timestamp" would be reasonable names.  I prefer blob since when it is read it might or might not be a timestamp.  The rest of the class is what makes it meaningful as a timestamp.